src/frontend/Komponenter/Behandling/SettPåVent/SettPåVentUtils.ts

```diff
@@ -0,0 +1,11 @@
+import { BeskrivelseHistorikkInnslag } from './BeskrivelseHistorikk';
+
+export const splitBeskrivelser = (beskrivelse: string): BeskrivelseHistorikkInnslag[] => {
+    return beskrivelse.split('\n\n').map((entry) => {
+        const [header, ...details] = entry.split('\n');
```

Comment:
Jeg tror, dessverre, ikke at beskrivelser alltid passer inn i formatet overskrift og detaljer etter to linjeskift. Saksbehandlere kan skrive fritt i beskrivelsesfeltet, og da kan du få mange pussige deler som kan oppleves som "overskrift", men bare er starten på et avsnitt.

Reply:
Fjernet helt nå, bra catch og enig! Vi kjører samme som i `ef-sak-frontend `for nå.